### DISCUSSION TITLE
help in evaluating the quality of my patches

### DISCUSSION THREAD

**Discussion by uazo** [Eloston/ungoogled-chromium#2396] [General]
ciao a tutti,

let me introduce myself, i am uazo and i have been collaborating with carl in the development of bromite for a while.
over time I have developed a number of patches, some simple, some somewhat more complex, for which, first with carl and now alone, design and technical choices have been discussed.
unfortunately, I am now alone and feel the lack of technical discussion.

the interest regarding my repo of late is making me think more carefully about whether I should broaden the scope of my work. can a single man go on? and also, is it okay to be alone?
i'm starting to explore some possibilities for collaboration in the android area, but i'm completely uncovered on my other (maybe bigger) interest which is the desktop side, which, because of my job, i use from morning to night.

I do not know the roles, skills and responsibilities of each person related to this project, so I am not in a position to address the question directly to the correct people; I do not even know if it is in your best interest to waste your time with this claptrap.

and so... i am here to ask if you are interested. in what? unfortunately i cannot express it definitively, i am a very (maybe too much :) reflective person and i like confrontation.

I know that, for now, one of the limitations is the total disorganization of my repo (after all I am italian :) but also the choice of license type but I think we can find the right solution for these too.

thank you for your attention.

**Comment by PF4Public** [comment]
Hi @uazo. I'm glad to see you around here!

I have considered pulling in Bromite patches in ungoogled-chromium since long time ago, but unfortunately we haven't figured out how we could integrate those patches and at the same time be compliant to Bromite licensing terms. So I doubt this could happen in immediate future, but maybe some day…

On the other hand it would be totally legal for anyone to use your and Bromite patches while building Chromium and ungoogled-chromium in particular locally for oneself. It is in fact what I do for quite some time. 

Recently I took the initiative and implemented a possibility for Gentoo users of ungoogled-chromium to conditionally apply a number of your patches while building ungoogled-chromium. At present only [a selection of patches is available](https://github.com/PF4Public/gentoo-overlay/blob/60663f2729547b7be0b4f5ff3a4c3a5d63b3380b/www-client/ungoogled-chromium/ungoogled-chromium-114.0.5735.133_p1.ebuild#L421-L451). There is also [a message text](https://github.com/PF4Public/gentoo-overlay/blob/60663f2729547b7be0b4f5ff3a4c3a5d63b3380b/www-client/ungoogled-chromium/ungoogled-chromium-114.0.5735.133_p1.ebuild#L317-L324) which could (I hope) encourage feedback :)

>can a single man go on? and also, is it okay to be alone?

That depends on targets you set up front. ungoogled-chromium initially was a one-man project of Eloston with casual contributions from others, but at some point it grew so big that one person could barely handle it :) See for yourself, what are your goals and what amount of time you have to spare for reaching those goals.

I'll ping @Ahrotahn as this discussion may be of interest in some regards.

**Reply by uazo** [reply]
thank you @PF4Public,
I'm not worried about changing the type of licence, I'm more worried about causing damage with my patches, considering that (it seems) many are moving from bromite to my version.
That is why I hope that they can also be checked and verified by others, because I enjoy to develop them and I would like to continue to do so

**Reply by Ahrotahn** [reply]
I don't mind looking over them as I get the time, though I lack experience with the java side of the project.  This is one of the reasons we had archived the android repo.

**Reply by uazo** [reply]
> I don't mind looking over them as I get the time

thank you in advance for your availability.

> I lack experience with the java side of the project.

Well, ehm... before working with Carl, I didn't even know Java... and I didn't even use android... (and, to tell the truth, my c++ was the scholastic one), so I think I have only to learn
